docs/MODULES.md

```diff
@@ -0,0 +1,150 @@
+| 67:68 | UDP     | Accept | DHCP Client/Server        |
+| 22    | UDP     | Accept | SSH                       |
+| 53    | TCP/UDP | Accept | DNS Server                |
+| 80    | TCP     | Accept | HTTP                      |
+| 443   | TCP     | Accept | SSH                       |
+| 9100  | TCP     | Accept | NodeExporter              |
+| 9090  | TCP     | Accept | Prometheus Server         |
+| 3000  | TCP     | Accept | Grafana                   |
+| 5201  | TCP     | Accept | IPerf3                    |
+| 4001  | TCP     | Accept | IPFS Swarm port           |
+
+#### Change open ports
+
+To change the open ports you can edit the IPv4 configuration file located at `/etc/iptables/rules.v4`
+
+Remove or comment out (`#`) the lines of the ports that you wish to close and add new lines for additional ports you wish to open.
+
+Likely you will not need to edit this file as much, if you are interested in opening mesh ports look at the **IPv6** section directly below.
+
+These are standard `iptables` rules. The basic syntax is as follows:
+
+`-A INPUT -j ACCEPT -p <protocol> --dport <port>`
+
+`protocol` -  either `tcp` or `udp`, not required but recommended
+
+`port` - Port you wish to open between 1-65535
+
+
+### IPv6
+
+Default open ports to the device over IPv6 are
+
+| Port | Protocol | Policy | Description               |
+| :---- | :------ | :----- | :------------------------ |
+| 67:68 | UDP     | Accept | DHCP Client/Server        |
+| 22    | UDP     | Accept | SSH                       |
+| 53    | TCP/UDP | Accept | DNS Server                |
+| 80    | TCP     | Accept | HTTP                      |
+| 443   | TCP     | Accept | SSH                       |
+| 9100  | TCP     | Accept | NodeExporter              |
+| 5201  | TCP     | Accept | IPerf3                    |
+| 4001  | TCP     | Accept | IPFS Swarm port           |
+
+#### Change open ports
+
+To change the open ports you can edit the IPv6 configuration file located at `/etc/iptables/rules.v6`
+
+Remove or comment out (#) the lines of the ports that you wish to close and add new lines for additional ports you wish to open.
+
+These are standard `ip6tables` rules. The basic syntax is as follows:
+
+`-A <table> -j ACCEPT -p <protocol> --dport <port>`
+
+`table` - `CJDNS` or `YGGDRASIL` for opening the port to CJDNS or YGGDRASIL, `YGGCLIENT` for opening up access to [**Yggdrasil Clients**](#yggdrasil-clients), and `INPUT` to open the port up to all of IPv6.
+
+`protocol` -  either `tcp` or `udp`, not required but recommended
+
+`port` - Port you wish to open between 1-65535
+
+Make sure to put your rules in the right section of the file, there are different ones depending on the table, with comments defining each section.
+
+#### Yggdrasil Clients
+Below are some different scenarios for opening up Yggdrasil clients. You will need to put these rules in `/etc/iptables/rules.v6`, in the Yggdrasil client rules section indicated by a comment.
+
+ - **One client, one port**
```

Comment:
I thought it would cover questions people might have about how to do it, explaining why it's a bad idea like you explained to me. It's not a big issue to remove it, but I think it just covers all our bases.